api/v1beta1/disruption_cron_webhook.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+package v1beta1
+
+import (
+	"errors"
+
+	"github.com/DataDog/chaos-controller/utils"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+var (
+	disruptionCronWebhookLogger     *zap.SugaredLogger
+	disruptionCronWebhookRecorder   record.EventRecorder
+	disruptionCronWebhookDeleteOnly bool
+)
+
+func (d *DisruptionCron) SetupWebhookWithManager(setupWebhookConfig utils.SetupWebhookWithManagerConfig) error {
+	disruptionCronWebhookRecorder = setupWebhookConfig.Recorder
+	disruptionCronWebhookDeleteOnly = setupWebhookConfig.DeleteOnlyFlag
+	disruptionCronWebhookLogger = setupWebhookConfig.Logger.With("source", "admission-controller")
```

Comment:
can we distinguish between the disruption webhook logger and the disruption cron webhook logger sources?